terraform/environments/staging/portal.tf

```diff
@@ -263,7 +263,7 @@ locals {
     # Auth
     {
       name  = "AUTH_PROVIDER_ADAPTERS"
-      value = "email,openid_connect,google_workspace,token,microsoft_entra,okta"
```

Comment:
Let's add it to `production/portal.tf` right away?

Reply:
Will that cause an issue if the adapter is not deployed?

Reply:
Nope, it's a whitelist: https://github.com/firezone/firezone/blob/andrew/bugfixes-11/elixir/apps/domain/lib/domain/auth/adapters.ex#L33